CRM/Upgrade/Incremental/Base.php

```diff
@@ -182,6 +182,51 @@ public static function addColumn($ctx, $table, $column, $properties, $localizabl
   }
 
   /**
+   * Modify a column in a table
+   *
+   * @param CRM_Queue_TaskContext $ctx
+   * @param string $table
+   * @param string $column
+   * @param string $properties
+   * @param bool $localizable is this a field that should be localized
+   * @param string $newColumn (new name for column)
+   * @return bool
+   */
+  public static function alterColumn($ctx, $table, $column, $properties, $localizable = FALSE, $newColumn = NULL) {
+    if (!isset($newColumn)) {
+      // Option to rename column
+      $newColumn = $column;
+    }
+    $domain = new CRM_Core_DAO_Domain();
+    $domain->find(TRUE);
+    $queries = array();
+    if (!CRM_Core_BAO_SchemaHandler::checkIfFieldExists($table, $column)) {
```

Comment:
So the difference here is that unlike with addColumn we need to check that the original column exists in the db. In regards to the Show it looks like it may translate the column name properly but we should probably add in tests here https://github.com/civicrm/civicrm-core/blob/b8fdc7b55458438c9e60008b954b0f9243db0388/tests/phpunit/CRM/Core/BAO/SchemaHandlerTest.php#L150 to verify that it does the appropriate re-writing